recipes-kernel/linux/linux-iot2050_5.x-ti-kernel.bb

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) Siemens AG, 2020
+#
+# Authors:
+#  Jan Kiszka <jan.kiszka@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+require recipes-kernel/linux/linux-custom.inc
+
+SRC_URI += " \
+    https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/snapshot/ti-linux-kernel-${KERNEL_REV}.tar.xz;protocol=https; \
+    file://ti-kernel/0001-iot2050-add-iot2050-platform-support.patch \
+    file://ti-kernel/0002-Add-support-for-U9300C-TD-LTE-module.patch \
```

Comment:
In my opinion, the word `ti-kernel` does not express the our exact intent very clearly.

Reply:
it clearly express the intents, it based ti kernel.
or What better advice do you have?

Reply:
We already have a ti-kernel targeting at 4.19.94. The target of this work is for the sr2 soc, so I think `sr2` would be a better name, or something else that help people understand this is for sr2.